util/math_reference.h

```diff
@@ -249,7 +249,9 @@ template <typename T>
 T bitselect(T a, T b, T c) {
   return (c & b) | (~c & a);
 }
+#if SYCL_CTS_ENABLE_HALF_TESTS
 sycl::half bitselect(sycl::half a, sycl::half b, sycl::half c);
+#endif
```

Comment:
Probably we should re-group these declarations to
```c++
// float type overloads for all math functions.
...

#if SYCL_CTS_ENABLE_DOUBLE_TESTS
// double type overloads for all math functions.
...
#endif

#if SYCL_CTS_ENABLE_HALF_TESTS
// half type overloads for all math functions.
...
#endif
```

Reply:
It would be great. Thanks!

Reply:
Done.